Tests/scripts/hook_validations/docker.py

```diff
@@ -0,0 +1,239 @@
+from Tests.test_utils import get_yaml, print_error, server_version_compare
+from pkg_resources import parse_version
+from datetime import datetime, timedelta
+import re
+import requests
+
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+ACCEPT_HEADER = {
+    'Accept': "application/json, "
+              "application/vnd.docker.distribution.manifest.v2+json, "
+              "application/vnd.docker.distribution.manifest.list.v2+json"
+}
+
+# use 10 seconds timeout for requests
+TIMEOUT = 10
+DEFAULT_REGISTRY = "registry-1.docker.io"
+
+
+def parse_www_auth(www_auth):
```

Comment:
why all of those functions are not in the class?

Reply:
They have no relations with the object, they are outer functions. Didn't thought they are needed to be inside as methods of the objects. Should they be inside?